h5/h5.ino

```diff
@@ -1790,15 +1797,26 @@ long getStepMaxSpeed(Axis* a) {
   return isContinuousStep() ? a->speedManualMove : min(long(a->speedManualMove), abs(getMoveStepForAxis(a)) * 1000 / STEP_TIME_MS);
 }
 
-void waitForStep(Axis* a) {
+void waitForStep(Axis* a, bool isMPG = false) {
```

Comment:
Is there a reason for MPG wait to be non-continuous? Seems like we just need a separate function `void waitForMpgStep(Axis* a)` and `waitForStep(Axis* a)` can stay unchanged.